src/overview/tokenomics.md

```diff
@@ -53,8 +62,11 @@ A user can check their current token balance in any of the following two ways:
 
 ## Current conversion rate
 
-- **Uploading fraud events**: `10` tokens per `id` fraud identifier.
-- **Downloading fraud events**: `1` token per `id` fraud identifier.
+| **Activity**              | **Reward**                     | **Cost**                     |
```

Comment:
I believe the table formatting should be based on tabulation to be straight
not 
```
   |
      |
  |
```

but
```
        |
        |
        |
```

Reply:
Edited